src/main/javascript/crypto/src/eas/EasTicketAttestation.ts

```diff
@@ -436,7 +442,7 @@ export class EasTicketAttestation extends AttestableObject implements Attestable
 
 	private processKeysParam(keys?: KeysArray){
 
-		let conferenceId = this.getAttestationField("devconId");
+		let conferenceId = this.getAttestationField("devconId", true);
```

Comment:
@micwallace , same here, I cant get logic why its better to have fallback to key ""(empty string)? thanks